content/10.diagnostics.md

```diff
@@ -244,28 +245,27 @@ They rely on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
+The authors showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, and pyrophosphates) that can all be used to detect SARS-CoV-2 RNA.
 Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.
 
 This goal of amplification-free detection was later achieved for a Cas13a-based system [@doi:10.1016/j.cell.2020.12.001].
 This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter.
 Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result.
 The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics.
 This approach is a truly portable assay for point-of-care diagnostics.
-The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n=20).
+The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n = 20).
 
 There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
 In the largest study to date, DETECTR was compared to RT-qPCR on 378 patient samples [@doi:10.1093/infdis/jiaa641].
 The authors reported 95% reproducibility.
 Both techniques were equally sensitive in detecting SARS-CoV-2.
 Lateral flow strips showed 100% correlation to the high-throughput DETECTR assay.
 Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
-A method based on a Cas9 ortholog from _Francisella novicida_ (FnCas9) achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].
+A method based on a Cas9 ortholog from _Francisella novicida_ known as FnCas9 achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].
 
 #### Immunoassays for the Detection of Antigens
 
 Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
-<!-- Move this: Since 2020, many immunoassays have been developed for COVID-19 [@url:https://www.centerforhealthsecurity.org/covid-19TestingToolkit/serology/Serology-based-tests-for-COVID-19.html] using several different technologies.-->
```

Comment:
This is a note I made to myself during revision that I don't think we need to listen to